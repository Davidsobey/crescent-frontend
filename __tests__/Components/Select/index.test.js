import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import configureStore from 'redux-mock-store'; // Smart components

// Component to be tested
import Select from '../../../src/Components/Select/index.js';


const mockStore = configureStore();
const initialState = {};
const store = mockStore(initialState);

describe('<Select />', () => {
  describe('render()', () => {
    test('renders the component', () => {
      const wrapper = shallow(<Select store={store} label="Course Name" input={{name:"courseId"}} meta={{touched:false}}  />);
      const component = wrapper.dive();

      expect(toJson(component)).toMatchSnapshot();
    });
  });
});