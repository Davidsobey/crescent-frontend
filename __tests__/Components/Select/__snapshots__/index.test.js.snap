// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Select /> render() renders the component 1`] = `
<FormControl
  classes={
    Object {
      "fullWidth": "MuiFormControl-fullWidth-4",
      "marginDense": "MuiFormControl-marginDense-3",
      "marginNormal": "MuiFormControl-marginNormal-2",
      "root": "MuiFormControl-root-1",
    }
  }
  component="div"
  disabled={false}
  error={false}
  fullWidth={false}
  margin="none"
  required={false}
>
  <styled.div>
    <WithStyles(InputLabel)
      htmlFor="courseId"
    >
      Course Name
    </WithStyles(InputLabel)>
    <WithStyles(Select)
      input={
        <WithStyles(Input)
          id="courseId"
        />
      }
    />
  </styled.div>
</FormControl>
`;
